test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py

```diff
@@ -51,11 +63,14 @@ def test_create_wallet(self):
                 crypto_client.create_wallet(alice_keys, "Alice" + str(validator_id))
                 with client.create_subscriber("blocks") as subscriber:
                     subscriber.wait_for_new_event()
-                self.assertEqual(crypto_client.get_wallet_info(alice_keys).status_code, 200)
+                self.assertEqual(
+                    crypto_client.get_wallet_info(alice_keys).status_code, 200
+                )
                 # TODO: Sometimes it fails without time.sleep() [ECR-3876]
                 time.sleep(2)
-                alice_balance = (crypto_client.get_wallet_info(alice_keys).json()
-                ['wallet_proof']['to_wallet']['entries'][0]['value']['balance'])
+                alice_balance = crypto_client.get_wallet_info(alice_keys).json()[
```

Comment:
Strange formatting.

Reply:
Formatting by `black`. What's wrong with it ?

Reply:
I agree that it looks very weird (seems that `black` isn't prepared for such a complex cases). Maybe split in several lines?

```python
wallet_info = crypto_client.get_wallet_info(alice_keys).json()
wallet_entry = wallet_info["wallet_proof"]["to_wallet"]["entries"][0]
alice_balance = wallet_entry["value"]["balance"]
```
?
```